sdk/deviceupdate/iot-device-update/src/operations/devices.ts

```diff
@@ -708,13 +684,10 @@ export class Devices {
     groupId: string,
     options?: coreHttp.OperationOptions
   ): Promise<DevicesGetGroupResponse> {
-    const { span, updatedOptions } = createSpan(
-      "DeviceUpdateClient-getGroup",
-      coreHttp.operationOptionsToRequestOptionsBase(options || {})
-    );
+    const { span, updatedOptions } = createSpan("DeviceUpdateClient-getGroup", options);
```

Comment:
Without this I was getting:

```
src/operations/devices.ts(1101,7): error TS2559: Type 'RequestOptionsBase' has no properties in common with type '{ tracingOptions?: OperationTracingOptions | undefined; }'.
```

Which I have to still figure out why the sudden difference